public/js/ticTacToe.js

```diff
@@ -3,6 +3,61 @@ const playerTurn = 'X'
 const Xmoves = []
 const Omoves = []
 
-const boardGame = document.querySelector('.board')
+const boardGame = document.querySelector('#board')
 const resetButton = document.querySelector('#reset')
+const divBoxes = document.querySelectorAll('div.box')
+const winningCombos = [[1,2,3], [4,5,6], [7,8,9], [1,4,7], [1,5,9], [3,5,7], [3,6,9], [2,5,8]]
 
+const switchTurn = function() {
+  if(playerTurn === 'X') {
+    playerTurn = 'O'
+  } else {
+    playerTurn = 'X'
+  }
+}
+
```

Comment:
Looks good. Not that anything should be changed, but just wanted to throw these your way so you can see a few alternatives. (They're not necessarily better or worse, just different)

Other ways of writing this:
1: Self calling object literal
```
const switchTurn = function() {
  playerTurn = { O: 'X', X: 'O' }[playerTurn]
}
```
Or perhaps:
2. Ternary Operation
```
const switchTurn = function() {
  playerTurn = playerTurn === 'X' ? 'O' : 'X'
}
```